framework/test/unit/modules/random/method.spec.ts

```diff
@@ -341,9 +340,9 @@ describe('RandomModuleMethod', () => {
 			);
 		});
 
-		it('should return XOR random bytes for height=11, numberOfSeeds=3', async () => {
+		it('should return XOR random bytes for height=11, numberOfSeeds=4', async () => {
```

Comment:
This test looks identical to the test of line 365. Am I missing something?

Reply:
yea it seems to be the same, the intended difference is this test doesn't include the invalid seed reveal and other one should include the invalid one, thus it has different outcome

Reply:
Good catch, @gkoumout. After updating the tests which were failing due to the protocol changes, the two test cases you are referring became identical by coincidence 😄 

I have updated one of the test, to validate random bytes when one seed reveal is invalid.